src/engine/qcommon/common.cpp

```diff
@@ -576,10 +576,9 @@ Com_Crash_f
 A way to force a bus error for development reasons
 =================
 */
-static void NORETURN Com_Crash_f()
+static void Com_Crash_f()
 {
 	* ( volatile int * ) 0 = 0x12345678;
```

Comment:
maybe document what's this line is while you're at it?

Reply:
Crashing of course :)

Reply:
Ah, well, yes didn't noticed the *0, I was focused on the 0x12346.... silly me